keyboards/rominronin/tsuka60/rev1_pro/readme.md

```diff
@@ -0,0 +1,28 @@
+# rominronin/tsuka60/rev1_pro
+
+![rominronin/tsuka60/rev1_pro](https://i.imgur.com/SLQ8bmY.jpg)
```

Comment:
```suggestion
![rominronin/tsuka60/rev1_pro](https://i.imgur.com/SLQ8bmYh.jpg)
```

Reply:
I assume the h suffix is a size optimisation step?

The URL doesn't work on desktop - it only seems to work on mobile - so I would revert this change.

If size is an issue, please say - I would then upload a smaller version.

Reply:
The modified link with the `h` in it works fine for me, both desktop and mobile 😅

Reply:
It still doesn't work for me. I pasted the link directly into a browser in incognito mode on each of Chrome, Firefox and Safari on Mac. The non-modified link works as expected.

What is the modified link supposed to be for? I'd be happy to follow your lead if it would work universally, but it doesn't work for me on my Mac.

Do you think the filesize is too big - is that it? Because I could just upload a smaller version...

Reply:
Tried it on an iPad too, and my Linux laptop on Chromium, 404 each time.

Reply:
It might have to do with your network rules or something.
Chrome Win 11:
![image](https://github.com/qmk/qmk_firmware/assets/40441626/8b7443de-967d-460a-8dda-6d0fa5fc5613)

Discord reader on iPhone:
![image](https://github.com/qmk/qmk_firmware/assets/40441626/4fbb64df-0631-46f7-aadd-b74a288078e3)

Chrome MacOS:
![image](https://github.com/qmk/qmk_firmware/assets/40441626/a778afc5-96dc-4962-b39d-da9d8b2b76ea)

Directly clicking on the link seems to have the 404 error, so that's more of a redirect thing on imgur side of things.
Copy-pasting for me seems to work. If you copy and paste in incognito 1 time it 404, but pasting it again and sending it works. So it's definitely a response thing caused by imgur if it 404.

Reply:
Ok - I'm happy to go with the suggestion - but since I cannot inspect the image, I need you to tell me WHY we are making this change.

There is also an Imgur image in the Katana60 Pro readme, why is that one ok?

Reply:
> I need you to tell me WHY we are making this change.

Maybe an "I would like you to explain to me why this change was suggested" or "Could you please provide some insight into the reasons behind this change?" could have been used here.
Anyway, the addition of `h` in Imgur links makes large images render at a lower resolution with the same aspect ratio.
This is to offload the unnecessary render time and size while loading a simple readme.
The issues with loading the image are hit or miss so that by itself isn't a reason to invalidate the usage of said flag. Even in the VSCode preview, the image loads fine most of the time and at times it doesn't, this is not relegated to the images in question but to others too, pointing once again at Imgur being Imgur and doing Imgur stuff.

> There is also an Imgur image in the Katana60 Pro readme, why is that one ok?

Either that one slipped through or wasn't noticed.

Reply:
Understood - I've commited the change